main.ts

```diff
@@ -369,7 +378,7 @@ function getHtmlPath(anyOsPath: string): string {
  */
 function setGlobalsFromVhaFile(vhaFileContents: FinalObject) {
   globals.hubName = vhaFileContents.hubName,
-  globals.screenshotSettings = vhaFileContents.screenshotSettings;
+    globals.screenshotSettings = vhaFileContents.screenshotSettings;
```

Comment:
Not your fault, but you highlighted what looks like a bug! 😄 
Can you change line 380 to have a `;` not `,`, and undo the formatting? 👍

Reply:
Done 😄 have you some configuration for formatting? Maybe will be useful if you write it to the .editorconfig... sorry for formatting :(

Reply:
Hmm, that's a good point - @whyboris do you want to decide on a consistent format, and add it to the repo? (eg. whether or not to have a space before the `()` in functions)

Reply:
I think we usually have no space before `()` in functions. Please feel free to add this rule wherever appropriate, I suspect it's one of these two? `.editorconfig` or `tslint.json` though perhaps it's better somewhere else?

ps -- looks like `tslint` is no longer maintained and we are encouraged to migrate to `eslint` -- so that could be a future PR 👍

Reply:
I'd recommend the `.editorconfig` unless we get CI at some point.

Reply:
Either way, I think that's for a new PR, and could be combined with running the auto formatter thought the rest of the code once the correct `.editorconfig` is setup that matches the current code style. 😄 

Resolving this issue. Thanks Erik! 👍